raiden/tests/integration/test_matrix_transport.py

```diff
@@ -110,6 +110,50 @@ def mock_receive_message(klass, message):
     return transport
 
 
+def ping_pong_message_success(transport0, transport1):
+    queueid0 = QueueIdentifier(
+        recipient=transport0._raiden_service.address,
+        channel_identifier=CHANNEL_IDENTIFIER_GLOBAL_QUEUE,
+    )
+
+    queueid1 = QueueIdentifier(
+        recipient=transport1._raiden_service.address,
+        channel_identifier=CHANNEL_IDENTIFIER_GLOBAL_QUEUE,
+    )
+
+    received_messages0 = transport0._raiden_service.message_handler.bag
+    received_messages1 = transport1._raiden_service.message_handler.bag
+    number_of_received_messages0 = len(received_messages0)
+    number_of_received_messages1 = len(received_messages1)
+
+    message = Processed(message_identifier=number_of_received_messages0)
+    transport0._raiden_service.sign(message)
+
+    transport0.send_async(queueid1, message)
+    with Timeout(20, exception=False):
+        all_messages_received = False
+        while not all_messages_received:
+            all_messages_received = (
+                len(received_messages0) == number_of_received_messages0 + 1 and
+                len(received_messages1) == number_of_received_messages1 + 1
+            )
+            gevent.sleep(.1)
+    message = Processed(message_identifier=number_of_received_messages1)
```

Comment:
Same as in #3385: This looks to me as if an `assert all_messages_received` is missing.

Reply:
This is supposed to happen in the tests as [here](https://github.com/raiden-network/raiden/pull/3598/files#diff-620b4fcbc4c15ae714616d075ea0b133R849), I found it cleaner to `assert` directly in the tests, not in the helper function